### PR TITLE
neuron features in morph_stats

### PR DESCRIPTION
**Pull request by arnaudon**
The features in the 'neuron' entry of the config dict will not be run on each neurite_type, but on the whole neuron only once. This avoid duplicating features if they are the same on all neurite types, such as the ones in neuronfunc.py

The returned dataframe is now of the form:
```  
       | property  | axon                  | all                   | neuron
       | name      | feature_1 | feature_2 | feature_1 | feature_2 | feature_3
   -------------------------------------------------------------------------------------
    0  | simple    | 0.1       | 0.2       | 0.15      | 0.25      | 1000
```

### CODE REVIEW

File: neurom/apps/morph_stats.py [hunk 1]
```diff
@@ -152,12 +139,13 @@ def extract_stats(neurons, config):
         neurons: a neuron, population, neurite tree or list of neuron paths/str
         config (dict): configuration dict. The keys are:
             - neurite_type: a list of neurite types for which features are extracted
-              If not provided, all neurite_type will be used
+              If not provided, all neurite_type will be used.
             - neurite: a dictionary {{neurite_feature: mode}} where:
-                - neurite_feature is a string from NEURITEFEATURES
+                - neurite_feature is a string from NEURITEFEATURES or NEURONFEATURES
```
Comment (wizmer) [inline neurom/apps/morph_stats.py:144]: not NEURONFEATURES

**Review by wizmer — COMMENTED** [review]
you need to update the `extract_dataframe` docstring as well

File: neurom/apps/morph_stats.py (hunk 1 above)
Comment (arnaudon) [reply neurom/apps/morph_stats.py:146]: both types of features works in both neurite or neuron entries. only the neuron entry will not aply to all neurite_types

**Comment by arnaudon** [comment]
not sure how to get the coverage up

**Comment by arnaudon** [comment]
ping @jdcourcol  :P

**Review by wizmer — APPROVED** [review]

**Comment by wizmer** [comment]
Thanks Alexis !

**Pull request merged**